src/test_l1_tvSettings.c

```diff
@@ -2219,7 +2218,7 @@ void test_l1_tvSettings_negative_SaveBacklight (void)
 	result = TvTerm();
 	UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);
 
-    videoFormat = UT_KVP_PROFILE_GET_UINT32("tvSettings/VideoFormat/index/0");
+	videoFormat = UT_KVP_PROFILE_GET_UINT32("tvSettings/VideoFormat/index/0");
```

Comment:
Line 2221 - videoFormat = UT_KVP_PROFILE_GET_UINT32("tvSettings/VideoFormat/index/0");

Is this really required? Already we have assigned the data in the start of the function.

Reply:
Just to make sure we are usign the index/0 of video format as we arae reading rthrough value in line 2203

Reply:
2171 and 2194- two times reinitialized, videoSource is not overwritten between these

Reply:
at 2202, the videoFormat is being is read from list.